src/__tests__/pageObject/pageObject.test.js

```diff
@@ -27,6 +31,21 @@ const templateTestData = [
   },
 ];
 
+const templateTestDataWithFindBy = [
+  {
+    input: "HTML",
+    output: pageObjectHTMLWithFindBy,
```

Comment:
one library would be enough. cases with different libraries are already covered

Reply:
more tests are better than fewer tests

Reply:
tests also needs support. as any duplicated code, they will need a twice more support

Reply:
Yes, I'm agree - it's a good point. 

Therefore, in the event of minimizing the count of unit and integration tests, we would consequently be placing the responsibility for error detection onto manual testers. 
Would this alternative be better for our project?

Reply:
@MariiaNebesnova  https://github.com/jdi-testing/jdn-ai/pull/1442/commits/83a1ecd87689743fa399fcdd17925932541b1f3e  deleted redundant code